app/views/kudo_mailer/batch_kudo_notification.text.erb

```diff
@@ -1,8 +1,9 @@
 <% content_for :message do %>
-<% @commentables.each_with_index do |commentable, index| %>
-<% givers_list = @kudo_givers["#{commentable.class.name}_#{commentable.id}"] %>
-<%= givers_list.each_with_index.map {|g,i| (g == "a guest" && i == 0) ? g.capitalize : g}.to_sentence.html_safe %> left kudos on "<%= commentable.commentable_name.html_safe %>" (<%= polymorphic_url(commentable) %>).<% if (index < @commentables.length-1) %>
-
-<%= text_divider %>
-
-<% end %><% end %><% end %>
\ No newline at end of file
+  <% @commentables.each_with_index do |commentable, index| %>
+    <% givers_list = @kudo_givers["#{commentable.id}"] %>
+    <%= if !givers_list.nil?  then givers_list.each_with_index.map {|g,i| ( i == 0) ? g.capitalize : g}.to_sentence end %> <%= t 'mailer.kudos.leftkudos' %> "<%= commentable.commentable_name.html_safe %>" (<%= polymorphic_url(commentable) %>).
```

Comment:
I didn't spot these last time because Github wraps this long line awkwardly, so here are some new comments:
1. there are two spaces after `!givers_list.nil?` - you only need one
2. the same unbalanced brackets around the `( i == 0)`